packages/gettext.rb

```diff
@@ -3,18 +3,18 @@
 class Gettext < Autotools
   description 'GNU gettext utilities are a set of tools that provides a framework to help other GNU packages produce multi-lingual messages.'
   homepage 'https://www.gnu.org/software/gettext/'
-  version "0.24-#{CREW_ICU_VER}"
+  version "0.25-#{CREW_ICU_VER}"
   license 'GPL-3+ and LGPL-2.1+'
   compatibility 'all'
-  source_url "https://ftpmirror.gnu.org/gettext/gettext-#{version.split('-')[0]}.tar.lz"
-  source_sha256 'caa44aed29c9b4900f1a401d68f6599a328a3744569484dc95f62081e80ad6cb'
+  source_url "https://ftp.gnu.org/pub/gnu/gettext/gettext-#{version.split('-')[0]}.tar.lz"
```

Comment:
The ftpmirror.gnu.org domain is not working?

Reply:
Might be ok, but source downloads happen so rarely that it should not be an issue.

In contrast the GNU Savannah git server always goes down ....